src/lib.rs

```diff
@@ -134,6 +142,10 @@ mod util;
 ///     `Some(...)`, relieving the caller from having to do this. Note that with this setting on
 ///     one cannot set the field to `None` with the setter - so the only way to get it to be `None`
 ///     is by using `#[builder(default)]` and not calling the field's setter.
+///
+///   - `transform = |params: Types, ...| expr`: this makes the setter accept `params: Types, ...`
+///     instead of the field type itself. The parameters are transformed into the field type using
+///     the expression `expr`.
```

Comment:
Since the transformation may have side-effects, I think it may be worthwhile to mention here that it is done when calling the setter (as opposed to when calling `build`)